boards/shields/waveshare_rs485_can/waveshare_rs485_can.overlay

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 Alexander Kozhinov
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,canbus = &arduino_can;
+	};
+};
+
+&arduino_serial {
+	status = "okay";
+	pinctrl-names = "default";
+};
+
+&arduino_can {
```

Comment:
What is `arduino_can`? Where it is defined? The Arduino Uno R3 pinout, which is what we typically use as shield reference in Zephyr, does not include any CAN RX/TX pins.

Reply:
> What is `arduino_can`? Where it is defined? The Arduino Uno R3 pinout, which is what we typically use as shield reference in Zephyr, does not include any CAN RX/TX pins.

Good point. I just named it similiar to `arduino_uart` like here https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/nucleo_h745zi_q/arduino_r3_connector.dtsi#L40
Speaking truly - I've never seen in zephyr something like `arduino_can` before. Please let me know, if you have other ideas for  naming.

Reply:
But how did you test it? Nothing in Zephyr defines this DTS node.

Reply:
> But how did you test it? Nothing in Zephyr defines this DTS node.

Thanks. It is valid. I've added definition to nucleo_h745zi_q board - shall be enough to for other developers to see how to define it.